File: Modul 2/Bahasa C (Modul 2)/PRAK204-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
#include <math.h>
int main(void){
    printf("input\n");
    int r,t;
    printf("");
    scanf("%d" "%d" , &r, &t);

    float phi = 3.14;
    float bejana = phi * r * r;
    float volume = phi * (r * r) * t;
    float keliling = 2 * phi * r;
    float luas = 2 * phi * r * ( r + t );

    printf("\nOutput\n");
    printf("Volume : %.2f\n", round(volume));
    printf("Luas : %.2f\n", round(luas));
    printf("Keliling : %.2f\n", round(keliling));

    return 0;
}

File: Modul 2/Bahasa Python (Modul 2)/PRAK204-2310817320015-AlfisyahRinaFajriati.py
#case 1
print("Input")
r = float (input(""))
t = float (input(""))

phi = 22/7
luas_bejana = phi*r
volume = phi*r*r*t
luas = 2*phi*r * (t + r)
keliling = 2*phi*r

print("\nOutput")
print(f"volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"keliling = {keliling:.2f}")

#case2
print("\nInput")
r,t = map(float, input().split())

phi = 22/7
luas_bejana = phi*r*r
volume = phi*r*r*t
luas = 2*phi*r*(r+t)
keliling = 2*phi*r

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
